modules/web-console/frontend/app/components/form-field/components/form-field-size/controller.ts

```diff
@@ -76,7 +76,16 @@ export default class PCFormFieldSizeController<T> implements IInputErrorNotifier
         if (!this.min) this.min = 0;
         if (!this.sizesMenu) this.setDefaultSizeType();
         this.$element.addClass('form-field');
-        this.ngModel.$render = () => this.assignValue(this.ngModel.$viewValue);
+        this.ngModel.$render = () => {
+            const rawValue = this.ngModel.$viewValue;
+
+            if (rawValue) {
+                this._sizeScale = _.findLast(this.sizesMenu,
+                    (val) => val.value <= rawValue && Number.isInteger(rawValue / val.value));
```

Comment:
Why do you pick last `sizesMenu` item instead of current `_sizeScale`? Why do you assign `_sizeScale` in `$render` at all if it should only be calculated based on bindings user user interaction when changing scale?

Reply:
Scale is selected on first rendering of field to show integer value if it is possible. Selected last approached scale to show.

Reply:
I don't think rendering callback should have anything with scale selection, it breaks single responsibility principle. Scale change is already handled by `$onChanges`, `$onInit` and `sizeScale` setter.

Then there's an issue where you randomly decide to pick last scale, ignoring all existing code that already does that. Or did I miss something?

Reply:
Since @akuznetsov-gridgain said that I have final authority on frontend matters at the moment, I'd like all out of scope features removed, especially changes to default scale choice algorithm.

Reply:
Removed auto detection of field scale.